.github/workflows/nightly.yml

```
@@ -72,15 +69,23 @@ jobs:
     - name: Setup environment
       run: .github/workflows/setup.sh
 
-    - name: Run tests
-      id: tests
-      run: .github/workflows/tests_nightly.sh
+    - name: Run pytest nigthly tests
+      id: pytest-tests
+      env:
+        AIIDA_TEST_PROFILE: test_aiida
+        AIIDA_WARN_v3: 1
+      run: |
+        pytest --db-backend psql -m nightly tests/
+
+    - name: Run daemon nightly tests
+      id: daemon-tests
+      run: source .venv/bin/activate && .github/workflows/daemon_tests.sh
```

```suggestion
      run: .github/workflows/daemon_tests.sh
```

I thing the venv is already activated by setup-uv action

---

Should be, changed.